app/audit/models/ChargeSummaryAudit.scala

```diff
@@ -34,7 +34,9 @@ case class ChargeSummaryAudit(mtdItUser: MtdItUser[_],
                               paymentAllocations: List[PaymentHistoryAllocations],
                               isLatePaymentCharge: Boolean,
                               isMFADebit: Boolean = false,
-                              taxYear: Int
+                              taxYear: Int,
+                              reviewAndReconcileEnabled: Boolean,
```

Comment:
this is the first ever audit event which we driver/execute by providing a feature switch status, why we need to pass it to audit event level?

Reply:
This audit model uses ChargeItem.getChargeTypeKey for some of the content, which takes feature switches. Otherwise if we have the reviewAndReconcile/penalties FS turned on and go to the Charge Summary for one of those types of charges, the audit will say "unknown charge"

Reply:
I think these charges would not be available on relevant page based on the FS i, so I would think that these check / flags not required at all